metrics/prometheus/prometheus.go

```diff
@@ -182,6 +184,11 @@ func NewMetrics(cfg config.PrometheusMetrics, disabledMetrics config.DisabledMet
 		"Count of total requests to Prebid Server labeled by type and status.",
 		[]string{requestTypeLabel, requestStatusLabel})
 
+	metrics.requestsDebug = newCounter(cfg, reg,
+		"requests_debug",
+		"Count of total requests to Prebid Server that have debug enabled labled by type, status.",
+		[]string{requestTypeLabel, requestStatusLabel})
```

Comment:
I do not believe we need to have any labels on this metric. I personally don't see any value in the status label and only minimal value in the types label. I won't put up a fight it you want to keep the type label, but I'll need convincing on why this is important for the status label.

Reply:
Yeah I struggled to think of reasons to have the labels, but I initially just matched the format of `metrics.requests`. But I feel comfortable with removing both